Jenkinsfile

```diff
@@ -3729,8 +3731,8 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         model.data.validation_ds.micro_batch_size=1 \
         model.data.validation_ds.global_batch_size=4 \
         model.data.validation_ds.num_workers=0 \
-        model.data.validation_ds.file_names=[/home/TestData/nlp/megatron_sft/quarel.jsonl,/home/TestData/nlp/megatron_sft/trec.jsonl] \
```

Comment:
@aklife97 thanks for the review! 
The issue here is Lightning 2.0 does not support multiple dataloaders. Lightning dropped the support in 2.0 since they thought it was too fragile in 1.9. I have asked lightning to provide a fix for this, since we consider it important. But until then it's a quick fix to disable the second validation dataset file here. Checked with MLPerf team and they are not using multiple dataloaders in their NeMo runs.

Reply:
Also this only when we are using `dataloader_iter`. 
[https://github.com/Lightning-AI/lightning/blob/master/src/lightning/pytorch/loops/evaluation_loop.py#L237](url)

Reply:
got it!